source/channel.c

```diff
@@ -864,12 +864,13 @@ int aws_channel_slot_shutdown(
     AWS_LOGF_TRACE(
         AWS_LS_IO_CHANNEL,
         "id=%p: shutting down slot %p, with handler %p "
-        "in %s direction with error code %d",
+        "in %s direction with error code %d (%s)",
         (void *)slot->channel,
         (void *)slot,
         (void *)slot->handler,
         (dir == AWS_CHANNEL_DIR_READ) ? "read" : "write",
-        err_code);
+        err_code,
+        aws_error_str(err_code));
```

Comment:
Personal preference, but I prefer `aws_error_name()` over `aws_error_str()`. But that's just me feel free to disagree

aws_error_name() will get you:

`id=0x12341234 shutting down slot 0x98769876, with handler 0x45674567 in read direction with error code  34 (AWS_ERROR_INVALID_ARGUMENT)`

aws_error_str(err_code) will get you:

`id=0x12341234 shutting down slot 0x98769876, with handler 0x45674567 in read direction with error code 34 (aws-c-common: AWS_ERROR_INVALID_ARGUMENT, An invalid argument was passed to a function.)`

Reply:
I second this, use either aws_error_name or debug_str